ci-operator/step-registry/baremetalds/two-node/arbiter/e2e-openshift-test-private-tests/baremetalds-two-node-arbiter-e2e-openshift-test-private-tests-workflow.yaml

```diff
@@ -16,6 +16,7 @@ workflow:
         NUM_MASTERS=2
         NUM_ARBITERS=1
         NUM_WORKERS=0
+        ARBITER_VCPU=2
```

Comment:
Is there also a MASTER_VCPU setting? Should we rely on the default, or nail it down to what we want (i think its 4)?